wolfcrypt/src/wc_port.c

```diff
@@ -124,7 +128,11 @@ int wolfCrypt_Init(void)
             return ret;
         }
     #endif
-    
+
```

Comment:
Should we also add a call to `IntelQaHardwareStop` in the cleanup function?

Reply:
We should be calling a couple other functions there, too.